web/app/routes/mainRoutes.js

```diff
@@ -0,0 +1,83 @@
+var express = require("express");
+var router = express.Router();
+var mongoose = require("mongoose");
+mongoose.connect("mongodb://localhost/routerdb");
+
+var userCtrl = require("../controllers/userCtrl");
+var drivingSchoolCtrl = require("../controllers/drivingSchoolCtrl");
+var drivingSessionCtrl = require("../controllers/drivingSessionCtrl");
+
+// server routes ===========================================================
+// handle things like api calls
+// authentication routes
+
+// Get the JSON for the student with the specified _id
+router.get("/students/:_id", function(req, res) {
+  userCtrl.getStudent(req, res);
```

Comment:
So you are proposing that I separate out the content checking of the request and move this logic up a level outside of the "createStudent" method?